pahfit/base.py

```diff
@@ -463,6 +469,19 @@ def save(self, obs_fit, filename, outform):
                     ]
                 )
             elif comp_type == "Drude1D":
+
+                # Calculate feature strength.
+                # TODO: Use input units to determine conversion and feature strength units.
+                strength = (np.pi * const.c.to('micron/s').value / 2) * (component.amplitude.value * component.fwhm.value / component.x_0.value**2) * 1e-26
+                # Estimate profile-weighted uncertainty.
+                if strength != 0.:
+                    nu = const.c.to('micron/s').value / x.value
+                    dnu = np.diff(nu, prepend=nu[1])
```

Comment:
Shouldn't this prepend nu[0]?

Reply:
It shouldn't make a difference if the element you are prepending is the 0th or 1st.

Reply:
Do I miss something? If you prepend the 0th, the first element in np.diff will be 0 while if you prepend the 1st, it will be (1st-0th)?

Reply:
Yes, that's the idea.